lib/timers.js

```diff
@@ -418,6 +404,27 @@ function processImmediate() {
 }
 
 
+function tryOnImmediate(immediate, queue) {
+  var threw = true;
+  try {
+    immediate._onImmediate();
+    threw = false;
+  } finally {
+    if (threw === true) {
```

Comment:
Hmmm I was under the impression we swallowed it for some reason, I guess this allows it to bubble up to the console?